packages/react-ui-core/src/Dropdown/Dropdown.js

```diff
@@ -79,12 +80,11 @@ export default class Dropdown extends Component {
           )}
         />
         {dropDownVisible &&
-          <div
+          <Card
             data-tid="dropdown-content"
```

Comment:
We seem to be mixed when it comes to naming this kind of thing. In some places its called Body, in others its called Content. I think most its Body, can we stay consistent?

Reply:
Sure, whatever we cant to call it is fine.  I'll change it to body.